piksi_tools/serial_link.py

```diff
@@ -95,6 +95,11 @@ def base_cl_options(override_arg_parse=None, add_help=True,
         "--verbose",
         action="store_true",
         help="print extra debugging information.")
+    parser.add_argument(
+        "-m",
+        "--metadata-skip",
```

Comment:
How about `--skip-metadata`

Reply:
That's probably better. Also I think the short switch can be left out.